web/src/stories/icon.stories.mdx

```diff
@@ -29,14 +29,13 @@ c.Config.Badge = component.BadgeDanger
           },
           config: {
             shape: "user",
-            size: "65",
+            size: "md",
             direction: "up",
             flip: "horizontal",
             solid: true,
             status: "danger",
             inverse: false,
             badge: "danger",
```

Comment:
Can you add badge color and label, we always try to include all properties (same for signpost).